lib/screens/MorePage.tsx

```diff
@@ -285,6 +285,12 @@ const styles = StyleSheet.create({
     height: undefined,
     resizeMode: "cover",
   },
+  lightModeText: {
```

Comment:
Nice and simple code here. Very understandable without the need to use comments.